modules/admin-ui/src/main/webapp/scripts/shared/partials/modals/delete-events-modal.html

```diff
@@ -60,166 +51,18 @@ <h2 translate="BULK_ACTIONS.DELETE.EVENTS.CAPTION"><!-- Delete Events --></h2>
             </div> <!-- obj -->
           </div><!-- full-col -->
 
-          <div class="full-col obj" ng-if="!events.unpublished.has">
-            <p translate="BULK_ACTIONS.DELETE.EVENTS.UNPUBLISHED.NONE">
```

Comment:
This PR should also remove the BULK_ACTIONS.* translation keys if they are not used anymore.

Reply:
Actually there still was one unused BULK_ACTIONS.* key which I just removed. The other BULK_ACTIONS.* keys are still in use.